applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.cpp

```diff
@@ -8,20 +8,166 @@
 //					 Kratos default license: kratos/license.txt
 //
 //  Main authors:    Philipp Bucher
+//                   Vicente Mataix Ferrandiz
+//                   Riccardo Rossi
+//                   Ruben Zorrilla
 //
 
 // System includes
 
 // External includes
 
 // Project includes
+#include "includes/checks.h"
 #include "structural_mechanics_element_utilities.h"
 #include "structural_mechanics_application_variables.h"
 #include "utilities/math_utils.h"
 
 namespace Kratos {
 namespace StructuralMechanicsElementUtilities {
 
+int BaseElementCheck(
+    const Element* pElement,
```

Comment:
```suggestion
    const Element& rElement,
```

From looking at the code you don't need the pointer